server/scripting/stubs/server.lua

```diff
@@ -0,0 +1,649 @@
+---    * - NUCLEAR_WINTER
+---      - Nuclear Winter is caused by the accumulation of nuclear fallout.
+edit.climate_change_type = {
+    GLOBAL_WARMING = "GLOBAL_WARMING",
+    NUCLEAR_WINTER = "NUCLEAR_WINTER"
+}
+
+--- Applies a wave of global climate change effects. Unlike regular climate
+--- change, this does not automatically generate any messages or affect the
+--- counters for pollution/fallout-related climate change. It will trigger even
+--- if :ref:`global_warming <server-option-global-warming>` or 
+--- :ref:`nuclear_winter <server-option-nuclear-winter>` options are disabled.
+---
+--- @param cctype edit.climate_change_type The type of climate change to apply.
+--- @param effect int The magnitude of the effect (approximately the number of tiles affected.)
+function edit.climate_change(cctype, effect) end
+
+--- Possibly throw a player into civil war. If the chance is not met, nothing
+--- happens. If the chance is met, civil war happens as usual, with appropriate
+--- messages sent to the players.
+---
+--- probability is the percentage chance of the civil war occurring (use 100
+--- for certainty), or if it is zero, the normal calculation is used (affected
+--- by government, happiness, etc).
+---
+--- Takes no account of the :ref:`server-option-civil-war-size` server option
+--- (but the player must have at least two cities).
+---
+--- @param player Player The player who may experience a civil war.
+--- @param probability int The probability of civil war occurring.
+--- @return Player rebel The new rebel AI player, or nil if it failed.
+function edit.civil_war(player, probability) end
+
+--- Change unit orientation to face in direction. 
+--- @param unit Unit The unit to change the orientation of.
+--- @param direction Direction The direction in which the unit will face.
+function edit.unit_turn(unit, direction) end
+
+--- Marks a player as victorious and ends the game.
+--- @param player Player The player to mark as the winner.
+function edit.player_victory(player) end
+
+--- Moves a unit to a new tile and deducts the cost from the units move points.
+--- If the tile is inhospitable terrain, or there are non-friendly units
+--- present, the unit dies. Very few checks are applied. Be careful!
+--- @param unit Unit The unit to be moved.
+--- @param move_to Tile The tile to move the unit to.
+--- @param move_cost int The cost of the move (in move fragments).
+--- @return bool survived True if the move was successful, false it died.
+function edit.unit_move(unit, move_to, move_cost) end
+
+--- Prohibits the unit from moving from the spot it's at.
+--- @param unit Unit The unit to disallow movement for.
+function edit.movement_disallow(unit) end
+
+--- Allows movement for a unit again following a 
+--- :lua:func:`edit.movement_disallow` call.
+--- @param unit Unit The unit to allow movement for.
+function edit.movement_allow(unit) end
+
+--- Adds history a.k.a. culture to a city. This affects civilisation score.
```

Comment:
I'm sure most people know AKA is also know as, but to ensure we don't cause any confusion you should add an entry to the glossary and then tag to it with a `:term:`